fog/view/enclave/api/src/lib.rs

```diff
@@ -123,6 +129,19 @@ pub trait ViewEnclaveApi: ReportableEnclave {
     /// Accept an inbound authentication request
     fn client_accept(&self, req: ClientAuthRequest) -> Result<(ClientAuthResponse, ClientSession)>;
 
+    /// Begin a connection to a Fog View Store. The enclave calling this method
```

Comment:
Nit: Keep `client_accept` and `client_close` adjacent.

Reply:
Done